cli/mne_bids_raw_to_bids.py

```diff
@@ -1,62 +1,58 @@
     parser.add_option('--task', dest='task',
-                      help='Name of the task the data is based on.',
-                      metavar='t')
+                      help='Name of the task the data is based on.')
     parser.add_option('--raw', dest='raw_fname',
-                      help='The path to the raw MEG file.', metavar='r')
+                      help='The path to the raw MEG file.')
     parser.add_option('--output_path', dest='output_path',
-                      help='The path of the BIDS compatible folder.',
-                      metavar='o')
+                      help='The path of the BIDS compatible folder.')
     parser.add_option('--session_id', dest='session_id',
-                      help='The session name in BIDS compatible format.',
-                      metavar='ses')
+                      help='The session name in BIDS compatible format.')
     parser.add_option('--run', dest='run',
-                      help='The run number for this dataset.',
-                      metavar='run')
+                      help='The run number for this dataset.')
     parser.add_option('--acq', dest='acq',
-                      help='The acquisition parameter.',
-                      metavar='acq')
+                      help='The acquisition parameter.')
     parser.add_option('--events_data', dest='events_data',
-                      help='The events file.', metavar='evt')
+                      help='The events file.')
     parser.add_option('--event_id', dest='event_id',
                       help='The event id dict.', metavar='eid')
     parser.add_option('--hpi', dest='hpi',
-                      help='The path to the MEG Marker points',
-                      metavar='filename')
+                      help='The path to the MEG Marker points')
     parser.add_option('--electrode', dest='electrode',
-                      help='The path to head-native Digitizer points',
-                      metavar='elec')
+                      help='The path to head-native Digitizer points')
     parser.add_option('--hsp', dest='hsp',
-                      help='The path to headshape points.', metavar='hsp')
+                      help='The path to headshape points.')
     parser.add_option('--config', dest='config',
-                      help='The path to the configuration file', metavar='cfg')
+                      help='The path to the configuration file')
     parser.add_option('--overwrite', dest='overwrite',
-                      help=("Boolean. Whether to overwrite existing data"),
-                      metavar='ow')
+                      help="Boolean. Whether to overwrite existing data")
     parser.add_option('--allow_maxshield', dest='allow_maxshield',
-                      help=("Boolean. Whether to allow non Maxfiltered data."),
-                      metavar='mxf', action='store_true')
+                      help="Boolean. Whether to allow non Maxfiltered data.",
+                      action='store_true')
```

Comment:
why did you remove the `metavar`. You shouldn't edit stuff not related to your PR.

Reply:
but what is the point of having a `metavar=r` for an argument `raw`? it is arguably more confusing to see the help showing `--raw=r` instead of `--raw=RAW`

going on along these lines, I got the idea that perhaps @teonbrooks mistook the `metavar` field for the "shortform" of an arg ... e.g., specify **RAW** either through `--raw` or `-r` ... however, to get that, you would need the following syntax (no `metavar`)

`parser.add_option('-r', '--raw', dest='raw', ...)`

I might be wrong though, and in that case - sorry for falsely suspecting Teon :-)

Reply:
That's fine, just do it in a different PR. That's all I am saying :-) Make life of reviewer easy ...